src/ziggurat/time.clj

```diff
@@ -0,0 +1,9 @@
+(ns ziggurat.time
```

Comment:
this namespace can also be moved into utils directory

Reply:
Namespace is changed